src/bee.ts

```diff
@@ -226,15 +252,17 @@ export class Bee {
   }
 
   /**
-   * Upload collection of files to a Bee node
+   * Upload collection of files.
    *
-   * Uses the `fs` module of Node.js
+   * Available only in NodeJs as it uses the `fs` module.
```

Comment:
On their website they use Node.js, I suggest we use it as well https://nodejs.org/en/
```suggestion
   * Available only in Node.js as it uses the `fs` module.
```